app/styles/layout/l-publications-articles.scss

```diff
@@ -0,0 +1,45 @@
+@import '../settings';
+
+.section-title {
+  margin-top: 0;
+}
+
+.l-publications-news {
+
+  .section-title {
+    margin: 0  0 30px;
+
+    @media #{$mq-tablet} {
+      margin: 0  0 60px;
+    }
+  }
+
+  section {
+    padding: 55px 0 65px;
+    position: relative;
+
+    &::after {
```

Comment:
Why not applying a border-bottom instead?
(you'd need to have `<div className={AppStyles.wrap}>` inside `section`s though)
Not sure about this.